### PR TITLE
Remove unsupported countries

### PR DESCRIPTION
**Pull request by islam3zzat**
#### Summary

In this PR we remove some countries that are no supported
We only support ISO 3166-1 country code.
See https://en.wikipedia.org/wiki/ISO_3166-1

Fixes: #1032

### CODE REVIEW

File: packages/l10n/data/countries/de.json [hunk 1]
```diff
@@ -40,13 +40,10 @@
   "ca": "Kanada",
   "cc": "Kokosinseln",
   "cd": "Kongo-Kinshasa",
-  "cd-alt-variant": "Kongo (Demokratische Republik)",
```
Comment (montezume) [inline packages/l10n/data/countries/de.json]: I wouldn't modify this directly as it's generated

File: packages/l10n/data/currencies/de.json [hunk 2]
```diff
@@ -181,7 +181,7 @@
   },
   "DZD": {
     "label": "Algerischer Dinar",
-    "symbol": "د.ج.‏"
+    "symbol": "DA"
```
Comment (emmenko) [inline packages/l10n/data/currencies/de.json]: Hmm why did those change? Did you install the latest deps?

Comment (islam3zzat) [reply packages/l10n/data/currencies/de.json]: I thought I did!
Updated the deps, and re-ran the `generate-data`

File: packages/l10n/data/time-zones/de.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
     "abbr": "-12",
     "offset": "-12:00"
   },
-  "Pacific/Niue": {
-    "name": "Pacific/Niue",
+  "Etc/GMT+11": {
```
Comment (islam3zzat) [inline packages/l10n/data/time-zones/de.json]: I'm not sure why this has changed.
But my deps are updated, and this was generated when I ran `generate-data`.
Will be very helpful if someone regenerated locally to make sure that this is how it is supposed to be.

Comment (emmenko) [reply packages/l10n/data/time-zones/de.json]: I regenerated the data. I think you might need to re-install all deps.

Comment (islam3zzat) [reply packages/l10n/data/time-zones/de.json]: Thanks a lot.
Will keep that in mind next time.

**Pull request merged**